elyra/pipeline/resources/palette.json

```diff
@@ -0,0 +1,158 @@
+{
+  "version": "3.0",
+  "categories": [
+    {
+      "label": "Notebook",
+      "image": "",
+      "id": "notebooks",
+      "description": "Notebook",
```

Comment:
After seeing the demo for Pipelines 2.0 in today's community meeting, should the icons for each of the palette entries be represented here?

Reply:
this is a copy of what is in master today, when we merge the new pipeline 2.0 there is going to be needed another integration pr, which can handle any additional requirements that comes from that pr.